tests/manager/manager-ctor-auto_encryption-001.phpt

```diff
@@ -2,6 +2,7 @@
 MongoDB\Driver\Manager::__construct(): auto encryption options
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_appveyor(); /* AppVeyor does not have mongocryptd installed */ ?>
```

Comment:
On travis-ci we install MongoDB Enterprise, and I thought I had changed that on appveyor as well. If we're not already installing the enterprise package, can you please create a tracking ticket for this?

Reply:
IIRC, AppVeyor never had `mongod` running at all. I don't see any reference to it in the [previous build scripts](https://github.com/mongodb/mongo-php-driver/tree/master/.appveyor). If you look back at our old builds I think you'll find that only tests with no server requirement were actually executed.

Reply:
Ah, yeah. That explains it 👍